dev/rabbitmq-config/definitions.json

```diff
@@ -24,14 +24,14 @@
     }],
     "policies": [],
     "queues": [{
-        "name": "my-queue",
+        "name": "openhpi-schulcloud-queue",
```

Comment:
If you define these in the config, why do you not use the config values here?

Reply:
This is the config for RabbitMQ and is only used in development. In order to use it, RabbitMQ needs some queues and exchanges configured. The app config is just adjusted to the existing RabbitMQ config. In production, there is already a configured RabbitMQ running somewhere.